### DISCUSSION TITLE
How to create orders with gross amount as 0 (zero) ?

### DISCUSSION THREAD

**Discussion by manojLondhe** [saleor/saleor#10045] [Q&A]
Hi,

We have a use case where we want create a sale under which we are offering discount which effectively ends up products having gross amount as 0 (zero).

Adding such products into a order (order created using Saleor admin dashboard) and placing order ends up showing payment status as `Fully Refunded`.

What is the correct way of creating 0 value orders?

**Comment by timuric** [comment]
@manojLondhe thanks for reporting, indeed it is missing we are going to fix it soon

**Reply by manojLondhe** [reply]
Thank you for confirmation @timuric 

Pls do consider all cases that make effective gross amount as 0 such as 
  - product is under a sale - effective price is 0
  - voucher applied - effective price is 0
  - discount applied when creating order from saleor-admin-dashboard - effective price is 0

**Reply by manojLondhe** [reply]
Should I report as bug under issues?

**Reply by timuric** [reply]
@manojLondhe yes, we should display Paid for 0 price orders

**Reply by manojLondhe** [reply]
yup, correct!
